Formula/fish.rb

```diff
@@ -22,9 +22,16 @@ class Fish < Formula
 
   def install
     system "autoconf" if build.head? || build.devel?
+    args = %W[
+      --prefix=#{prefix}
+      --with-extra-functionsdir=#{HOMEBREW_PREFIX}/share/fish/vendor_functions.d
```

Comment:
@zanchey Great to hear. soft-failing does sound sane.
it works on my system.
```
/usr/bin/install -c -m 755 -d /usr/local/share/fish/vendor_completions.d || true
install: chmod 755 /usr/local/share/fish/vendor_completions.d: Operation not permitted
/usr/bin/install -c -m 755 -d /usr/local/share/fish/vendor_functions.d || true
install: chmod 755 /usr/local/share/fish/vendor_functions.d: Operation not permitted
/usr/bin/install -c -m 755 -d /usr/local/share/fish/vendor_conf.d || true
install: chmod 755 /usr/local/share/fish/vendor_conf.d: Operation not permitted
```

@DomT4 It does not put anything, just creates empty dirs  and teaches fish to look there
 ```
$(INSTALL) -m 755 -d $(DESTDIR)$(extra_completionsdir)
$(INSTALL) -m 755 -d $(DESTDIR)$(extra_functionsdir)
$(INSTALL) -m 755 -d $(DESTDIR)$(extra_confdir)
```



can I just stop makefile from installing the dirs and mkpath instead? seems to be the cleanest solution. fish upstream can definitely soft fail extra dir creation and we'll create in our desired location ourselves. I've pushed another commit with above changes, not sure how to ask Jenkins to re-run the check.